app/jobs/assignment_repo/porter_status_job.rb

```diff
@@ -10,6 +10,8 @@ class PorterStatusJob < ApplicationJob
     def perform(assignment_repo, user)
       github_repository = assignment_repo.github_repository
 
+      invite_status = assignment_repo.assignment.invitation&.status(user)
```

Comment:
Same comment here as above. If there is no invitation, can we just abort the job?